### PR TITLE
BZ-1869593: Recommend against modifying the default roles

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://bugzilla.redhat.com/show_bug.cgi?id=1869593

Preview: https://deploy-preview-34988--osdocs.netlify.app/openshift-enterprise/latest/authentication/using-rbac.html#default-roles_using-rbac

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@barleyer Can you please QE review this update?

Preview: https://deploy-preview-34988--osdocs.netlify.app/openshift-enterprise/latest/authentication/using-rbac.html#default-roles_using-rbac

**Comment by bergerhoffer** [comment]
Just heard that shengsheng left Red Hat :(. 

@xltian @rhpmali @yaoli-redhat Can you help me with who the right QE person is for this PR? I see a new name next to RBAC in the team spreadsheet, but couldn't find them in rover or GitHub.

File: modules/rbac-overview.adoc [hunk 1]
```diff
@@ -79,8 +79,12 @@ For example:
 == Default cluster roles
 
 {product-title} includes a set of default cluster roles that you can bind to
-users and groups cluster-wide or locally. You can manually modify the default
-cluster roles, if required.
+users and groups cluster-wide or locally.
```
Comment (jeana-redhat) [inline modules/rbac-overview.adoc:82]: Could unwrap this if you're so inclined

**Review by jeana-redhat — APPROVED** [review]
LGTM

**Comment by yaoli-redhat** [comment]
@bergerhoffer yash is not onboard yet. One concern is why is it allowed to modify if 'Modifications to these system roles can prevent a cluster from functioning properly.'?

**Comment by xingxingxia** [comment]
/lgtm
> why is it allowed to modify

Cluster admins have the power to do things.

**Pull request merged**